ui/src/present.ts

```diff
@@ -0,0 +1,38 @@
+// Copyright © 2021 The Radicle Upstream Contributors
```

Comment:
What do you think about calling this module `format`?

Reply:
My initial thinking was that the function calls would read more naturally, like this: `present.ethAddress`. I guess I'm still stuck in ruby presenter pattern land.

Renamed as per your suggestion, so it's less confusing: ede1ef8fc2128e8df4f4e717bfc45bdead41ed68.